### ISSUE TITLE
proguard configuration

### ISSUE THREAD

**Issue by DJafari** [iammert/MaterialIntroView#3]
thanks for your great library !

can you post proguard configuration please ?

i give this warning now :

```Warning:co.mobiwise.materialintro.view.MaterialIntroView$$Lambda$1: can't find referenced class java.lang.invoke.LambdaForm$Hidden
Warning:co.mobiwise.materialintro.view.MaterialIntroView$$Lambda$2: can't find referenced class java.lang.invoke.LambdaForm$Hidden
Warning:co.mobiwise.materialintro.view.MaterialIntroView$$Lambda$3: can't find referenced class java.lang.invoke.LambdaForm$Hidden
Warning:co.mobiwise.materialintro.view.MaterialIntroView$1$$Lambda$1: can't find referenced class java.lang.invoke.LambdaForm$Hidden
Warning:co.mobiwise.materialintro.view.MaterialIntroView$1$$Lambda$4: can't find referenced class java.lang.invoke.LambdaForm$Hidden```

**Comment by iammert** [comment]
Ok I fixed it. Can you check it again. Use version 1.2.

**Comment by DJafari** [comment]
thanks
